server/routes/Project.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+
+const { projectById, getUserProjects, imageUpload, addProject } = require('../controllers/project')
+const { verifyJwt } = require('../middlewares')
+const auth = require('../controllers/authController.api');
+const { isAuth, userById } = auth;
+
+router.get('/projects/:userId', getUserProjects);
+router.post('/projecs/image-upload', imageUpload);
+router.post('/projects/add/:userId', verifyJwt, isAuth, addProject);
```

Comment:
can we just combine the verifyJwt and isAuth?

Reply:
I think we can do that! I'll refactor it once I'm home later today